dhis-2/dhis-services/dhis-service-reporting/src/main/java/org/hisp/dhis/interpretation/impl/DefaultInterpretationService.java

```diff
@@ -107,6 +115,8 @@ public int saveInterpretation( Interpretation interpretation )
             }
 
             interpretation.updateSharing();
+            
+            interpretation.setMentions(this.updateMentions( interpretation.getText() ));
```

Comment:
This works on creation, but on update I don't see the _mentions_ field updated, I think a similar line is a missing `updateInterpretation`

Reply:
Yep, I missed that one. Already fixed!